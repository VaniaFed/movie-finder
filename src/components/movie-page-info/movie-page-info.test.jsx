import renderer from 'react-test-renderer';
import { MoviePageInfo } from './index';

describe('<MoviePageInfo />', () => {
    const genres = ['action', 'drama'];
    const tree = renderer.create(<MoviePageInfo genres={genres} />).toJSON();
    it('should be rendered with genres', () => {
        expect(tree).toMatchSnapshot();
    });
});
