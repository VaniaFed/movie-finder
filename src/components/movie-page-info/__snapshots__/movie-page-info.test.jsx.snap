// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviePageInfo /> should be rendered with genres 1`] = `
<div
  className="movie-page-info"
>
  <span
    className="movie-page-info__text"
  >
    Films by
  </span>
  <span
    className="movie-page-info__genre"
  >
    action, drama
  </span>
</div>
`;
